src/User/index.ts

```diff
@@ -1,11 +1,12 @@
 import { favoritesCache, tripsDetailedCache, userCache } from '../Cache';
 import { reset as resetChanges } from '../Changes';
 import * as Dao from './DataAccess';
-import { UserSettings } from './User';
+import { ThirdPartyAuthType, UserSettings } from './User';
 import { UserSession } from './UserSession';
 import * as UserSessionManager from './UserSession';
 
 export {
+	ThirdPartyAuthType,
```

Comment:
Is this supposed to be also in `index.d.ts` ?

Reply:
No. On outher interface we have two separate methods